### PR TITLE
Demo of get Decision tree

### LINKED ISSUE #4: Decision tree: Demo model
A subtask of #3 
Build demo model of decision tree.

### PR DESCRIPTION
**Pull request by JOHNKYON**
Resolve #4.

The project structure also varied.

### CODE REVIEW

File: models_demo/tree/decision_tree.py [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+    # Remove numeric predictors
+    melb_numeric_predictors = melb_predictors.select_dtypes(exclude=['object'])
+
+    # Split data into training and validation data for both predictos and target
+    train_x, test_x, train_y, test_y = train_test_split(melb_numeric_predictors, melb_target,
+                                                        random_state=0)
+
+    # Imputating data
+
```
Comment (ZhuGongpu) [inline models_demo/tree/decision_tree.py]: Remove this line.

File: test/test_null.py [hunk 2]
```diff
@@ -0,0 +1,11 @@
+"""This is the test case that do nothing, it's just used to pass travis-CI when no tests are
+available"""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+
+def testnull():
```
Comment (ZhuGongpu) [inline test/test_null.py:9]: Why not remove this file?

File: models_demo/tree/decision_tree.py [hunk 3]
```diff
@@ -0,0 +1,47 @@
+"""Model demos of decision tree"""
```
Comment (ZhuGongpu) [inline models_demo/tree/decision_tree.py:1]: How about rename the `models_demo` into `examples`? @JOHNKYON

**Review by ZhuGongpu — CHANGES_REQUESTED** [review]

File: test/test_null.py (hunk 2 above)
Comment (JOHNKYON) [reply test/test_null.py:9]: If I remove this file, pytest will return a stupid code 5, that will never pass the travis-CI build check.

**Pull request closed without merging**